technical_docs/Claims.md

```diff
@@ -71,26 +141,37 @@ This is equivalent to the following URI-form:
 
```

Comment:
I'm confused about when we use the URI-form versus the equivalent short form. Why do we need both?